meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 # which must contain only alphanumeric symbols,'-' and '_'.
 # Any other symbols would be skipped in parser.
 #
-# Following targets would be used to perform default build task:
-OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
+# Following targets would be used to perform default build task.
+OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
```

Comment:
On Tue, 2016-03-29 at 01:21 -0700, Alexander D. Kanevskiy wrote:
> In meta-ostro/conf/distro/include/ostroproject-ci.inc:
> 
> > @@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
> >  # which must contain only alphanumeric symbols,'-' and '_'.
> >  # Any other symbols would be skipped in parser.
> >  #
> > -# Following targets would be used to perform default build task:
> > -OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
> > +# Following targets would be used to perform default build task.
> > +OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
> 
> I prefer to have ${DISTRO} here instead of hardcoded ostro.

That would have the drawback of not being able to grep for image names
when looking for them. For example, when I more recently replaced
ostro-image-qa with ostro-image-dev, I could use grep and be reasonably
sure I covered it.

I could imagine that someone might want to fork Ostro OS and then the
DISTRO variable would make some sense, but this is our Ostro OS CI
configuration - I just don't see that advantage outweighing the
disadvantage mentioned above.